libs/protocol-client/src/tests/utils/ipfs.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, expect, test } from '@jest/globals';
+import Ajv from 'ajv';
+import v1Schema from '../../schemas/contributionMetadataV1.json';
+import { MintedContributionSchemaV1 } from '../../lib/utils/ipfs';
+import { ContributionV1SchemaCid } from '../../lib/utils/constants';
+
+describe('IPFS test', () => {
+  test('test schema matches type', () => {
+    const ajv = new Ajv(); // options can be passed, e.g. {allErrors: true}
+    const validate = ajv.compile(v1Schema);
```

Comment:
Ah that's so cool!

Reply:
yea, if there is a mismatch we probably need a new schema